szymonwietowski/Reservation
# Touk Recruitment

Requirements:
- scala
- sbt
- curl
- jq

Testing steps:
1) Run the start.sh file and wait for the application to start.
2) Run demo.sh and follow the steps to make reservation.
   To make another reservation run demo.sh again. 
   If server error raised, then close and start demo app to start process from beginning.
3) To check reservations run 'http://localhost:9000/reservations' in browser.
